admin/scripts/image-resize.mjs

```diff
@@ -25,7 +25,7 @@ await Promise.all(
       new URL(`../../website/src/data/showcase/${img}`, import.meta.url),
     );
     const {width, height} = imageSize(imgPath);
-    if (width === 640 && height === 320) {
+    if (width === 640 && height === 320 && imgPath.endsWith('.png')) {
```

Comment:
what's the purpose of this new condition?

Reply:
We normalize JPEG images to PNG, but if an image has 640×320 dimensions it will be ignored and stay as JPEG.